content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
-## Manubot Rootstock Information
+## Introduction
 
-**Note: Manubot instances should delete this file.**
+This manuscript is a template/boilerplate (aka "rootstock") for [Manubot](https://manubot.org/ "Manubot"), a tool for writing scholarly manuscripts.
+Use this template as a starting point for your manuscript by following the directions on the [repository homepage](https://github.com/greenelab/manubot-rootstock "Manubot Rootstock").
 
-The Manubot is a system for automating scholarly publishing.
-Content is written in [Pandoc Markdown](http://pandoc.org/MANUAL.html#pandocs-markdown) source files.
-See [`USAGE.md`](https://github.com/greenelab/manubot-rootstock/blob/master/USAGE.md) for more information on how to use the Manubot.
 
-The Manubot project began with the [Deep Review](https://github.com/greenelab/deep-review), where it was used to compose a highly-collaborative review article [@doi:10.1098/rsif.2017.0387].
-Another example manuscript that was created with Manubot is:
 
-+ The Sci-Hub Coverage Study ([GitHub](https://github.com/greenelab/scihub-manuscript), [HTML manuscript](https://greenelab.github.io/scihub-manuscript/)) [@doi:10.7554/eLife.32822]
+## Examples
 
-If you notice a problem with Manubot, it's best to submit an upstream fix to the appropriate repository:
-[`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) for the git repository stub or [`greenelab/manubot`](https://github.com/greenelab/manubot) for the Python package.
+Below is a full list of formatting elements for `.html` and `.pdf` exports that are supported by the Manubot-provided themes. 
+Compare the input Markdown (`.md` files in the `/content` directory) to the output you see below.
+
+# Heading 1
+
+## Heading 2
+
+### Heading 3
+
+#### Heading 4
+
+##### Heading 5
```

Comment:
Not sure if we should show h5 and h6. Although they are supported by the theme and markdown, It's likely the user is making a mistake by using them. Sometimes users use heading format to make things the size they want / bolded, without regard to the hierarchical structuring of headers.

Reply:
This is useful for separating content & styling. Other formats might make use of additional headers, so it's still useful to write with `####` and `#####` (think about a book that might have have a part, chapter, section, and subsection: 1.1.1.1).

Reply:
What I should've said is I don't know why they included `h5` and `h6` and made them smaller than the normal font size, if they're intended to represent hierarchical structure. Perhaps we should just make them all bigger than the normal font size then and keep them in.